isisd/isis_zebra.h

```diff
@@ -35,16 +35,13 @@ void isis_zebra_init(struct thread_master *master, int instance);
 void isis_zebra_stop(void);
 
 struct isis_route_info;
-struct sr_prefix;
 
 void isis_zebra_route_add_route(struct prefix *prefix,
 				struct prefix_ipv6 *src_p,
 				struct isis_route_info *route_info);
 void isis_zebra_route_del_route(struct prefix *prefix,
 				struct prefix_ipv6 *src_p,
 				struct isis_route_info *route_info);
-void isis_zebra_install_prefix_sid(const struct sr_prefix *srp);
-void isis_zebra_uninstall_prefix_sid(const struct sr_prefix *srp);
```

Comment:
what was the motivation to move the SR zapi code out of this module?

Reply:
There is two SR Zapi code: one to install/uninstall SR prefix-SID in `isisd/isis_zebra.c` file and another one to install/uninstall SR Adjacency-SID in `isisd/isis_sr.c` file. For coherency, I think it is preferable to group both piece of code into the same file. I chosen `isisd/isis_sr.c` file to take benefit of `sr_debug` macro, but I could group them into `isisd/isis_zebra.c`.

Reply:
One function computes the input/output labels and the other one actually sends the Prefix-SID install message to zebra. I think the latter belongs in _isisd/isis_zebra.c_, as that file contains all functions related to communication with the zebra daemon.

Reply:
> One function computes the input/output labels and the other one actually sends the Prefix-SID install message to zebra.

All functions that manage LFIB installation for Prefix-SID and Adjacency-SID do more or less the same things: fulfil the zapi_labels structure and call zebra_send_mpls_labels() function.
So, I don't understand why they are not group in the same file. I choose isis_sr.c to take benefit of sr_debug() macro, but, if you think it is more appropriate to move them into isis_zebra.c, not problem. I just would avoid keeping them separate between Adjacency and Prefix SID.

Reply:
Yes, please move it back.

> I just would avoid keeping them separate between Adjacency and Prefix SID.

I didn't realize `sr_adj_sid_install_uninstall()` was part of _isis_sr.c_. That function also belongs in _isis_zebra.c_.

Reply:
Done